inode/config.go

```diff
@@ -116,6 +121,8 @@ func Up(confMap conf.ConfMap) (err error) {
 		flowControl                                    *flowControlStruct
 		flowControlName                                string
 		flowControlSectionName                         string
+		LRUCacheMaxBytes                               uint64
+		LRUDiscardTimeInterval                         time.Duration
```

Comment:
As we've discussed, I dislike any sort of polling and would much prefer triggering the code that evicts at the point of insertion. Sure, it could be done in the background (by simply launching a goroutine)... indeed, that is the pattern for lots of things going on in ProxyFS (check out the quota limiting the number of chunked PUTs underway). Sure, it's a bit complicated to manage, but is far more deterministic than polling which could fall behind and is, thus, difficult to tune.

What I'm suggesting is we have a (lock-protected) flag that says whether a eviction goroutine is already running or not such that when inserting in the inodeCache would trigger an eviction that a new goroutine is only launched when that flag is false. Hope this makes sense.